arangod/Aql/QueryList.cpp

```diff
@@ -208,8 +196,8 @@ void QueryList::remove(Query* query) {
         // free first element
         _slow.pop_front();
       }
+    } catch (...) {
```

Comment:
Just need to ignore the erorr here is the right thing?

Reply:
Yes. There is not much we can do here if inserting the query into the list of slow queries.
It is possible that this fails due to not being able to acquire memory for it. In case it fails, the slow list is not modified and nothing is leaked. But there is nothing else we could do except logging a message. But I am not sure if that is worth it.